frontend/src/pages/GraphPF/MiniGraphCardPF.tsx

```diff
@@ -213,6 +221,10 @@ class MiniGraphCardPFComponent extends React.Component<MiniGraphCardPropsPF, Min
     );
   }
 
+  private handleReady = (refs: GraphRefs): void => {
+    this.setState({ graphRefs: refs, isReady: true });
+  };
```

Comment:
I messed things up a bit when adding in the new `handleReady/onReady` logic, because I neglected to ensure we still dispatched the previous `onReady` call.  Having said that, I'm fairly sure that MiniGraphCardPF can remove the redux prop for onReady, and not worry about it.  That action is really relevant to initializing the side panel, and there isn't one in the minigraph.

What I think is missing is to add the call back in GraphPagePF#handleReady().